### PR TITLE
Quickfix

### PR DESCRIPTION
**Pull request by crowdedpoem**
Fixing bugs found in ingredient input and saving username for form

### CODE REVIEW

File: backend/models/recipe-services.js [hunk 1]
```diff
@@ -93,7 +95,7 @@ function standardize(num, unit) {
   var base = {
     tsp: "tsp",
     tbsp: "tsp",
-    "fl oz": "tsp",
+    floz: "tsp",
```
Comment (Tpanyasr) [inline backend/models/recipe-services.js:98]: thanks for catching this

File: backend/models/recipe.js [hunk 2]
```diff
@@ -64,12 +63,24 @@ const RecipeSchema = new mongoose.Schema(
     urlSource: {
       type: String,
       required: false,
+      trim: false,
+    },
```
Comment (Tpanyasr) [inline backend/models/recipe.js:67]: I think image and cost can be removed from the schema since they are generated from api call

**Review by Tpanyasr — CHANGES_REQUESTED** [review]
Nice job just left a couple comments about things to change

File: frontend/src/components/Form.js [hunk 3]
```diff
@@ -82,9 +80,10 @@ function Form() {
 
   function addStep() {
     // Generate a dynamic number of inputs
-    numSteps++;
```
Comment (tylerboo) [inline frontend/src/components/Form.js:85]: what does this numStep do, and do we use the value of this variable somewhere else?

File: frontend/src/components/Form.js [hunk 4]
```diff
@@ -229,6 +228,16 @@ function Form() {
...
 
   function handleChange(event) {
+    if (recipe.user === "") {
+      console.log("handle change is fixign user");
+      const currentUser = AuthService.getCurrentUser();
+      console.log(currentUser.username);
+      let another = recipe;
+      let name = String(currentUser.username);
+      another["user"] = name;
+      setRecipe(another);
```
Comment (tylerboo) [inline frontend/src/components/Form.js:238]: Is it possible to assign username to the recipe by calling setRecipe("user": currentUser.username)?

**Review by tylerboo — APPROVED** [review]
Looks good overall!

File: frontend/src/components/Form.js (hunk 3 above)
Comment (crowdedpoem) [reply frontend/src/components/Form.js:85]: Yeah this was a bug, it was supposed to keep track of the steps for the recipe but it wasn't saving state. Now we just check how many inputs with a class name "step" there are in the form

File: frontend/src/components/Form.js (hunk 4 above)
Comment (crowdedpoem) [reply frontend/src/components/Form.js:238]: Yes, that was my original attempt but it was buggy and didn't always work. This seems to do the trick

**Review by Tpanyasr — APPROVED** [review]
Looks good!

**Pull request merged**